Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM ubuntu:16.04
+
+RUN mkdir /almond
+COPY . /almond
+
+# Install sudo, git, curl, libpulse, and https sources support
+RUN apt-get update && apt-get -y install sudo
```

Comment:
You don't need sudo in a docker container, by default it runs as container root.